### PR TITLE
Added Commands

### PR DESCRIPTION
**Pull request by sachinl0har**
Added Country, Applestore and Playstore commands in the fun section as you said.

### CODE REVIEW

File: commands/fun/country.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		}
+		catch (e) {
+			return message.channel.send(
+				':x:An error occured, please try again!',
+			);
+		}
+		try{
+			const data = response[0];
+			const embed = new MessageEmbed()
+				.setColor('BLUE')
```
Comment (DeltaCoderr) [inline commands/fun/country.js:35]: Can you replace the Color of the embed to the one in the `config` for all these commands?. As that's the one the bot uses globally.

Comment (sachinl0har) [reply commands/fun/country.js]: Done

File: commands/fun/applestore.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+const Discord = require('discord.js');
+const config = require('../../configs/config.json');
+const AppleStore = require("app-store-scraper");
+const EmbedColor = `RANDOM`;
```
Comment (DeltaCoderr) [inline commands/fun/applestore.js:4]: Also this line, remove it.

Comment (sachinl0har) [reply commands/fun/applestore.js]: Done

**Review by DeltaCoderr — APPROVED** [review]

**Pull request merged**